bin/check-hardware-fail.rb

```diff
@@ -29,9 +29,26 @@
 require 'sensu-plugin/check/cli'
 
 class CheckHardwareFail < Sensu::Plugin::Check::CLI
+  option :lines,
+         short: '-l NUMBER',
+         long: '--lines NUMBER',
+         proc: proc(&:to_i),
+         default: 0,
```

Comment:
its fine just checking if there was something particularly clever that I had not seen before. I think operational code should be more than less clear.